docs/servers.rst

```diff
@@ -166,68 +147,53 @@ installation media (USB or CD) and press Enter to boot it.
-window will ask for your hostname. To get back to the choice of
-configuring the network manually, **Cancel** the step that asks you to
-set a hostname and choose the menu option that says **Configure the
-network manually** instead.
-
-For a production install with a pfSense network firewall in place, the
-*Application Server* and the *Monitor Server* are on separate networks.
-You may choose your own network settings at this point, but make sure
-the settings you choose are unique on the firewall's network and
-remember to propagate your choices through the rest of the installation
-process.
+The **Edit eno1 IPv4 configuration** dialog will be displayed. In the **IPv4 Method** menu, select **Manual**, then add your server-specific settings.
+
+.. note:: For a production install with a pfSense network firewall in place, the
+  *Application Server* and the *Monitor Server* are on separate networks.
+  You may choose your own network settings at this point, but make sure
+  the settings you choose are unique on the firewall's network and
+  remember to propagate your choices through the rest of the installation process.
 
 Below are the configurations you should enter, assuming you used the
 network settings from the network firewall guide for a 3 NIC or 4 NIC firewall.
 If you did not, adjust these settings accordingly.
 
 -  *Application Server*:
 
-  -  Server IP address: 10.20.2.2
-  -  Netmask (default is fine): 255.255.255.0
-  -  Gateway: 10.20.2.1
-  -  For DNS, use Google's name servers: 8.8.8.8 and 8.8.4.4
-  -  Hostname: app
-  -  Domain name should be left blank
+  -  **Subnet:** 10.20.2.0/24
+  -  **Address:** 10.20.2.2
+  -  **Gateway:** 10.20.2.1
+  -  **Name servers:** 8.8.8.8, 8.8.4.4
+  -  **Search Domains:** *should be left blank*
 
 -  *Monitor Server*:
 
-  -  Server IP address: 10.20.3.2
-  -  Netmask (default is fine): 255.255.255.0
-  -  Gateway: 10.20.3.1
-  -  For DNS, use Google's name servers: 8.8.8.8 and 8.8.4.4
-  -  Hostname: mon
-  -  Domain name should be left blank
-
-Continue the Installation
-~~~~~~~~~~~~~~~~~~~~~~~~~
+  -  **Subnet:** 10.20.3.0/24
+  -  **Address:** 10.20.3.2
+  -  **Gateway:** 10.20.3.1
+  -  **Name servers:** 8.8.8.8, 8.8.4.4
+  -  **Search Domains:** *should be left blank*
 
-You can choose whatever username and passphrase you would like. To make
-things easier later you should use the same username and same passphrase
-on both servers (but not the same passphrase as username). Make sure to
-save this passphrase in your admin KeePassXC database afterwards.
+Select **Save** and press **Enter** to apply your settings. Then select **Done** and press **Enter**.
 
-Click 'no' when asked to encrypt the home directory. Then configure your
-time zone.
+The default values on the **Configure Proxy** and **Configure Ubuntu archive mirror**
+screens should not need to be changed. Select **Done** for both.
 
-Partition the Disks
-~~~~~~~~~~~~~~~~~~~
+Full Disk Encryption - pros and cons
```

Comment:
I'm of the opinion that we should be a little firmer and more succinct in this section. Something like

> Full Disk Encryption
> 
> We discourage the use of  `Full Disk Encryption (FDE) <https://www.eff.org/deeplinks/2012/11/privacy-ubuntu-1210-full-disk-encryption>`__. with SecureDrop.  While FDE does offer data protection for devices that are powered down, SecureDrop's servers are designed to be always-on, with the exception of a nightly reboot after automatic upgrades are applied. 
> Given this update schedule, with FDE enabled, the servers would become unreachable once every 24 hours until an administrator entered the full-disk encryption passphrase via the console, and during that time, sources and journalists would be unable to access your instance. 
>   
> The increased responsibility for administrators, as well as the daily downtime and limited scenarios in which FDE would be a net security benefit, inform our recommendation, but you may make an informed decision based on your own requirements. (See `this GitHub
> issue <https://github.com/freedomofpress/securedrop/issues/511#issuecomment-50823554>`__
> for more information).

Reply:
(This isn't necessarily something you have to address in this particular PR if you want to leave it alone.)

Reply:
No I think this wording is better (absent the "we"s which this has also spurred me into changing throughout the doc).